micropip/package_index.py

```diff
@@ -153,6 +154,11 @@ def _compatible_wheels(
             hashes = file["digests"] if "digests" in file else file["hashes"]
             sha256 = hashes.get("sha256")
 
+            # Check if the metadata file is available (PEP 658 / PEP-714)
+            core_metadata: DistributionMetadata = file.get("core-metadata") or file.get(
+                "data-dist-info-metadata"
+            )
+
```

Comment:
Question: should this be `"data-core-metadata"` (might have been missed)? The keys for the HTML API are prefixed with `"data-"`, but those for the JSON API are not.
```suggestion
            # Check if the metadata file is available (PEP 658 / PEP-714)
            core_metadata: DistributionMetadata = file.get("data-core-metadata") or file.get(
                "data-dist-info-metadata"
            )
	
```

Reply:
Good question. The `mousevender` renames `data-core-metadata` into `core-metadata` when loading HTML to Python dict so that both JSON and HTML responses can access the same key.